collect/get_top_pypi.py

```diff
@@ -84,16 +89,20 @@ def get_package_stats(data_tasks, f):
 
 
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--max-repos", help="Maximum number of repos to get", type=int, default=5000)
+    args = parser.parse_args()
+
     # Start selenium driver to get top 5000 pypi page
     url_top_pypi = "https://hugovk.github.io/top-pypi-packages/"
     driver = webdriver.Chrome()
     driver.get(url_top_pypi)
-    button = driver.find_element(By.CSS_SELECTOR, 'button[ng-click="show(5000)"]')
+    button = driver.find_element(By.CSS_SELECTOR, 'button[ng-click="show(8000)"]')
```

Comment:
If we're adding the `--max-repos` arg, should this be `f'button[ng-click="show({args.max_repos})"]'` instead?

Reply:
Oh! I see, the [website](https://hugovk.github.io/top-pypi-packages/) says 8000 now, thanks for the catch, understood!